setup.py

```diff
@@ -23,171 +24,189 @@
 data files.
 """
 try:
-    from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
+    from wheel.bdist_wheel import bdist_wheel
 
-    class bdist_wheel(_bdist_wheel):
+    # noinspection PyClassicStyleClass,PyAttributeOutsideInit
+    class CustomBDistWheel(bdist_wheel):
         def finalize_options(self):
-            _bdist_wheel.finalize_options(self)
+            # noinspection PyArgumentList
+            bdist_wheel.finalize_options(self)
             self.root_is_pure = False
 except ImportError:
-    bdist_wheel = None
     print("setup requires that the Python package 'wheel' be installed. "
           "Try the command 'pip install wheel'.")
     sys.exit(1)
 
 
-def getExecutableDir():
+def get_executable_dir():
     """
     Build source path on the afdko for the command-line tools.
     """
-    curSystem = platform.system()
-    if curSystem == "Windows":
-        binDir = "win"
-    elif curSystem == "Linux":
-        binDir = "linux"
-    elif curSystem == "Darwin":
-        binDir = "osx"
+    cur_system = platform.system()
+    if cur_system == "Windows":
+        bin_dir = "win"
+    elif cur_system == "Linux":
+        bin_dir = "linux"
+    elif cur_system == "Darwin":
+        bin_dir = "osx"
     else:
         raise KeyError("Do not recognize target OS: %s" % sys.platform)
     platform_name = get_platform()
-    return binDir, platform_name, curSystem
+    return bin_dir, platform_name, cur_system
 
 
-def compile(pgkDir):
+def compile_package(pkg_dir):
     if 0:
         print("Skipping compile - hard coded.")
         return
-    binDir, platform_name, curSystem = getExecutableDir()
-    programsDir = os.path.join(pgkDir, "Tools", "Programs")
-    if binDir == 'osx':
+    bin_dir, platform_name, cur_system = get_executable_dir()
```

Comment:
Change to
```python
bin_dir, _, _ = get_executable_dir()
```

Reply:
I'm going to change this so `get_executable_dir()` only returns the executable dir, like it says in the name.